.vscode/snippets.code-snippets

```diff
@@ -0,0 +1,26 @@
+{
+	// Place your azure-sdk-for-c workspace snippets here. Each snippet is defined under a snippet name and has a scope, prefix, body and 
+	// description. Add comma separated ids of the languages where the snippet is applicable in the scope field. If scope 
+	// is left empty or omitted, the snippet gets applied to all languages. The prefix is what is 
+	// used to trigger the snippet and the body will be expanded and inserted. Possible variables are: 
+	// $1, $2 for tab stops, $0 for the final cursor position, and ${1:label}, ${2:another} for placeholders. 
+	// Placeholders with the same ids are connected.
+	// Example:
+	// "Print to console": {
+	// 	"scope": "javascript,typescript",
+	// 	"prefix": "log",
+	// 	"body": [
+	// 		"console.log('$1');",
+	// 		"$2"
+	// 	],
+	// 	"description": "Log output to console"
+	// }
+	"copyright and license header": {
+		"prefix": "copyright",
+		"body": [
+			"$LINE_COMMENT Copyright (c) Microsoft Corporation. All rights reserved.",
```

Comment:
Does this do the right thing for C? (`/* */`)

Reply:
I don't know what this does for C, but I know that C has had `//` comments for 20 years.

Reply:
Right, C99 minimum version :) Thanks for the note!